packages/react/src/app-layout/AppLayoutHeaderAccount.tsx

```diff
@@ -39,10 +46,12 @@ export function AppLayoutHeaderAccount({
 			alignItems="center"
 			flexShrink={0}
 			gap={0.5}
+			minHeight="5.25rem"
 			textAlign="right"
 			css={mq({
+				marginLeft: 'auto',
+				maxWidth: `calc(17.625rem - ${scrollbarWidthRef.current}px)`,
```

Comment:
What are the significances of these `rem` values?

Reply:
This is the maximum space available at 375px for it to sit alongside the Menu button

Reply:
Comment added